BambooTracker/res/lang/bamboo_tracker_fr.ts

```diff
@@ -394,22 +394,22 @@
     <message>
         <location filename="../../gui/configuration_dialog.cpp" line="281"/>
         <source>Reconstruct the current channel&apos;s state from previous orders upon playing.</source>
-        <translation type="unfinished"></translation>
+        <translation>Reconstruire l&apos;état du canal actuel à partir des ordres précédents durant la lecture.</translation>
```

Comment:
-> des canaux

Reply:
Why? there isn't plural in original text.

Reply:
Misread, my bad.